core/trino-main/src/main/java/io/trino/sql/planner/plan/SimpleTableExecuteNode.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.sql.planner.plan;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import io.trino.metadata.TableExecuteHandle;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.sql.planner.Symbol;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class SimpleTableExecuteNode
+        extends PlanNode
+{
+    private final Symbol output;
+    private final TableExecuteHandle executeHandle;
+    private final SchemaTableName schemaTableName;
+
+    @JsonCreator
+    public SimpleTableExecuteNode(
+            @JsonProperty("id") PlanNodeId id,
+            @JsonProperty("output") Symbol output,
+            @JsonProperty("executeHandle") TableExecuteHandle executeHandle,
+            @JsonProperty("schemaTableName") SchemaTableName schemaTableName)
+    {
+        super(id);
+        this.output = requireNonNull(output, "output is null");
+        this.executeHandle = requireNonNull(executeHandle, "executeHandle is null");
+        this.schemaTableName = requireNonNull(schemaTableName, "schemaTableName is null");
+    }
+
+    @Override
+    @JsonProperty
+    public List<PlanNode> getSources()
+    {
+        return ImmutableList.of();
+    }
+
+    @Override
+    public List<Symbol> getOutputSymbols()
+    {
+        return ImmutableList.of(output);
+    }
+
+    @JsonProperty
+    public Symbol getOutput()
+    {
+        return output;
+    }
+
+    @Override
+    public PlanNode replaceChildren(List<PlanNode> newChildren)
+    {
+        return new SimpleTableExecuteNode(getId(), output, executeHandle, schemaTableName);
```

Comment:
`newChildren` is ignored

add `checkArgument(newChildren.isEmpty())` since children are not expected
and then `return this;`

Reply:
I can do that but again I tried to follow the behaviour of `TableDeleteNode` 
https://github.com/trinodb/trino/blob/a57c6c769101e7aeb91efd61ab8130f30516aae5/core/trino-main/src/main/java/io/trino/sql/planner/plan/TableDeleteNode.java#L77-L80

Reply:
see `io.trino.sql.planner.plan.ValuesNode#replaceChildren`.
let's fix `TableDeleteNode`too

Reply:
ok!